catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/image/NitfImageTransformer.java

```diff
@@ -95,10 +101,20 @@ private void handleSegments(NitfSegmentsFlow nitfSegmentsFlow, Metacard metacard
             MultiPolygon multiPolygon = GEOMETRY_FACTORY.createMultiPolygon(polyAry);
             metacard.setAttribute(new AttributeImpl(Core.LOCATION, multiPolygon.toText()));
         }
+
+        // Set start, effective, and end from discovered imageDateAndTimes
+        if (!imageDateAndTimeList.isEmpty()) {
+            LOGGER.trace("Setting the {}, {}, and {} metacard attributes from the discovered imageDateTimes of the image segments.", Metacard.EFFECTIVE, ddf.catalog.data.types.DateTime.START, ddf.catalog.data.types.DateTime.END);
+            final Date firstDateAndTime = imageDateAndTimeList.get(0);
+            final Date lastDateAndTime = imageDateAndTimeList.size() > 1 ? imageDateAndTimeList.get(imageDateAndTimeList.size() - 1) : firstDateAndTime;
```

Comment:
If there is one image, then imageDateAndTimeList contains one element, so that 'imageDateAndTimeList.size() - 1' will be '0'. That gives the same result as firstDateAndTime. Granted, it is an extra call to 'get' when not absolutely needed, but it simplifies the code.